File: algorithms/golang/recurrent_seq.go
package main

import (
    "fmt"
)

func main() {
    xn := 2
    result := 2*xn + 10
    for result < 100 {
        xn = result
        result = 2*xn + 10
    }

    fmt.Printf( "Result: %d\n", result )
}

File: algorithms/cpp/recurrent_seq.cpp
#include <iostream>
using namespace std;

int main() 
{
    int xn = 2;
    int result = 2*xn + 10;

    while( result < 100 )
    {
        xn = result;
        result = 2*xn + 10;
    }

    cout << "Result: " << result << "\n";
}
